### PR TITLE
VZ 6904: Rancher SPDY TLS continuation

### PR DESCRIPTION
**Pull request by anders-swanson**
TLS termination was occurring at ingress-nginx, breaking SPDY connections.

### CODE REVIEW

**Review by davisgomes — DISMISSED** [review]

File: platform-operator/thirdparty/charts/rancher/templates/ingress.yaml [hunk 1]
```diff
@@ -48,10 +48,10 @@ spec:
           service:
             name: {{ template "rancher.fullname" . }}
             port:
-              number: 80
+              number: 443
```
Comment (bobdonat) [inline platform-operator/thirdparty/charts/rancher/templates/ingress.yaml:51]: This is going to problematic when we pick up a new Rancher chart/templates.

Comment (anders-swanson) [reply platform-operator/thirdparty/charts/rancher/templates/ingress.yaml:51]: I've created a PR to the upstream helm chart so the port can be configurable. In the meantime we will have this edit.

File: platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml [hunk 2]
```diff
@@ -276,7 +276,9 @@ spec:
             app.kubernetes.io/instance: ingress-controller
       ports:
         - protocol: TCP
-          port: 80
+          port: 443
+        - protocol: TCP
+          port: 444
```
Comment (bobdonat) [inline platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml:281]: What is port 444

Comment (davisgomes) [reply platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml:281]: I'm still not sure I understand its inclusion. I don't see this used in the e2e tests

Comment (anders-swanson) [reply platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml:281]: Rancher service uses 443:444

File: platform-operator/thirdparty/charts/rancher/templates/ingress.yaml (hunk 1 above)
Comment (anders-swanson) [reply platform-operator/thirdparty/charts/rancher/templates/ingress.yaml:51]: Once we take up rancher 2.7.x, it will be included there.

**Review by davisgomes — APPROVED** [review]

**Review by bobdonat — APPROVED** [review]

**Pull request merged**